Tools/ros2/ardupilot_msgs/srv/ModeSwitch.srv

```diff
@@ -0,0 +1,15 @@
+
+# This service requests the vehicle to switch its drive/flight mode
+
+# mode : Set the value to the drive/flight mode to be used
+# Copter : https://github.com/ArduPilot/ardupilot/blob/master/ArduCopter/mode.h#L14 
+# Rover  : https://github.com/ArduPilot/ardupilot/blob/master/Rover/mode.h#L12
+# Plane  : https://github.com/ArduPilot/ardupilot/blob/master/ArduPlane/mode.h#L22
```

Comment:
Is there a more maintainable way to link the modes that won't get out of date when the line numbers change?

Reply:
Perhaps we could just drop the line numbers and provide a link to respective wikis for more detail?